apps/opencs/view/doc/viewmanager.cpp

```diff
@@ -60,6 +61,40 @@ CSVDoc::ViewManager::ViewManager (CSMDoc::DocumentManager& documentManager)
     mDelegateFactories->add (CSMWorld::ColumnBase::Display_RefRecordType,
         new CSVWorld::IdTypeDelegateFactory());
 
+    // Columns with QLineEdit editor
+    static const CSMWorld::ColumnBase::Display sIdCompletionColumns[] =
+    {
+        CSMWorld::ColumnBase::Display_Cell,
+        CSMWorld::ColumnBase::Display_Class,
+        CSMWorld::ColumnBase::Display_Creature,
+        CSMWorld::ColumnBase::Display_Faction,
+        CSMWorld::ColumnBase::Display_String,
+        CSMWorld::ColumnBase::Display_GlobalVariable,
+        CSMWorld::ColumnBase::Display_Icon,
+        CSMWorld::ColumnBase::Display_Mesh,
+        CSMWorld::ColumnBase::Display_Miscellaneous,
+        CSMWorld::ColumnBase::Display_Music,
+        CSMWorld::ColumnBase::Display_None, // Inplace editing (Table SubView) creates QLineEdit using Display_None
+        CSMWorld::ColumnBase::Display_Npc,
+        CSMWorld::ColumnBase::Display_Race,
+        CSMWorld::ColumnBase::Display_Referenceable,
+        CSMWorld::ColumnBase::Display_Region,
+        CSMWorld::ColumnBase::Display_Script,
+        CSMWorld::ColumnBase::Display_Skill,
+        CSMWorld::ColumnBase::Display_Sound,
+        CSMWorld::ColumnBase::Display_SoundRes,
+        CSMWorld::ColumnBase::Display_Static,
+        CSMWorld::ColumnBase::Display_String,
```

Comment:
There is one small problem. This list overlaps with EnumDelegate display types. So creating factories for the IdCompletionDelegate must be located before EnumDelegate factories.